README.md

```diff
@@ -190,6 +190,7 @@ Every dataset and pipeline folder must contain a `dataset.yaml` and a `pipeline.
 
 # Best Practices
 
+- When naming BigQuery columns, always use snakecase and lowercase.
```

Comment:
Same feedback as https://github.com/GoogleCloudPlatform/public-datasets-pipelines/pull/6/files#r619378699 `snake_case` could be clearer.

Reply:
Resolved via #6.